perceval/rendering/pdisplay.py

```diff
@@ -320,10 +326,88 @@ def pdisplay_tomography_chi(qpt: ProcessTomography, output_format: Format = Form
     plt.show()
 
 
+def _complex_to_rgb(z: complex, cmap='hsv'):
+    """for better rendering, cmap should be a cyclic matplotlib ColorMap"""
+    r, g, b, a = colormaps[cmap]((phase(z) + pi) / (2 * pi))
+    a = abs(z)
+    vect = np.array([r, g, b])
+    vect = (a / norm(vect)) * vect
+    return vect
+
+
+def _csr_to_rgb(matrix, cmap='hsv'):
+    if matrix.ndim != 2:
+        raise ValueError(f"matrix should be a 2d array, not {matrix.ndim}d")
+
+    img = np.zeros(matrix.shape + (3,))
+    coef_max = 0
+    for i in range(matrix.shape[0]):
+        for j in range(matrix.shape[0]):
+            z = matrix[i, j]
+            if z != 0:
+                img[i, j, :] = _complex_to_rgb(z)
+                if abs(z) > coef_max:
+                    coef_max = abs(z)
+    img = (1/coef_max) * img
+    return img
+
+
+def _csr_to_greyscale(matrix):
+
+    if matrix.ndim != 2:
+        raise ValueError(f"matrix should be a 2d array, not {matrix.ndim}d")
+
+    img = np.zeros(matrix.shape)
+    coef_max = 0
+    for i in range(matrix.shape[0]):
+        for j in range(matrix.shape[0]):
+            z = matrix[i, j]
+            if z != 0:
+                img[i, j] = 255*abs(z)
+                if abs(z) > coef_max:
+                    coef_max = abs(z)
+    img = (1 / coef_max) * img
+    return img
+
+
+def generate_ticks(dm):
+    m, n = dm.m, dm.n_max
+    tick_list = [0]
+    tick_labels = ["0 photon"]
+    for k in range(n):
+        tick_list.append(math.comb(m+k, k))
+        tick_labels.append(str(k+1)+" photons")
+    return tick_list, tick_labels
+
+
+def pdisplay_density_matrix(dm,
+                            output_format: Format = Format.MPLOT,
+                            color: bool = True,
+                            cmap='hsv'):
+
+    if output_format == Format.TEXT or output_format == Format.LATEX:
+        raise TypeError(f"DensityMatrix plot does not support {output_format}")
+    if color:
```

Comment:
So `color` means you either display a colorful or a grayscale map. Ok. But, I don't understand what does the gray level of each pixel represents, in grayscale mode. The same but we lose the phase info?

Is `color` the correct name for this parameter?
At least add some docstring to explain what it does, please.